modules/asg/main.tf

```diff
@@ -92,7 +92,7 @@ resource "aws_autoscaling_group" "instance" {
 }
 
 resource "aws_lb_target_group" "instance" {
-  name                 = format("%s-tg-canary-%s", var.common.common_name, var.name)
+  name                 = format("%s-tg-can-%s", var.common.common_name, var.name)
```

Comment:
would dropping "tg-" be more informative than abbreviating "canary" to "can"?

Reply:
Good catch fixed now